shadowJar: Analyzing ORT with ORT fails

Reproduce e.g.:
```
./gradlew cli:shadowJar
java -cp cli/build/libs/cli-40deb7a7d5-dirty-all.jar org.ossreviewtoolkit.cli.OrtMainKt analyze -i . -o . -m Maven,Gradle
```

Outcome:

Exception in thread "main" org.codehaus.plexus.component.repository.exception.ComponentLookupException: java.util.NoSuchElementException
      role: org.eclipse.aether.RepositorySystem
  roleHint: default
	at org.codehaus.plexus.DefaultPlexusContainer.lookup(DefaultPlexusContainer.java:267)
	at org.codehaus.plexus.DefaultPlexusContainer.lookup(DefaultPlexusContainer.java:255)
	at org.ossreviewtoolkit.analyzer.managers.utils.MavenSupport.createRepositorySystemSession(MavenSupport.kt:873)
	at org.ossreviewtoolkit.analyzer.managers.utils.MavenSupport.<init>(MavenSupport.kt:321)
	at org.ossreviewtoolkit.analyzer.managers.Gradle.<init>(Gradle.kt:136)
	at org.ossreviewtoolkit.analyzer.managers.Gradle.<init>(Gradle.kt:74)
	at org.ossreviewtoolkit.analyzer.managers.Gradle$Factory.create(Gradle.kt:93)
	at org.ossreviewtoolkit.analyzer.managers.Gradle$Factory.create(Gradle.kt:83)
	at org.ossreviewtoolkit.analyzer.Analyzer.findManagedFiles(Analyzer.kt:91)
	at org.ossreviewtoolkit.cli.commands.AnalyzerCommand.run(AnalyzerCommand.kt:259)
	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:198)
	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:211)
	at com.github.ajalt.clikt.parsers.Parser.parse(Parser.kt:18)
	at com.github.ajalt.clikt.core.CliktCommand.parse(CliktCommand.kt:400)
	at com.github.ajalt.clikt.core.CliktCommand.parse$default(CliktCommand.kt:397)
	at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:415)
	at com.github.ajalt.clikt.core.CliktCommand.main(CliktCommand.kt:440)
	at org.ossreviewtoolkit.cli.OrtMainKt.main(OrtMain.kt:76)
Caused by: java.util.NoSuchElementException
	at org.eclipse.sisu.inject.LocatedBeans$Itr.next(LocatedBeans.java:141)
	at org.eclipse.sisu.inject.LocatedBeans$Itr.next(LocatedBeans.java:1)
	at org.eclipse.sisu.plexus.DefaultPlexusBeans$Itr.next(DefaultPlexusBeans.java:76)
	at org.eclipse.sisu.plexus.DefaultPlexusBeans$Itr.next(DefaultPlexusBeans.java:1)
	at org.codehaus.plexus.DefaultPlexusContainer.lookup(DefaultPlexusContainer.java:263)
	... 17 more

--- a/cli/build.gradle.kts
+++ b/cli/build.gradle.kts
@@ -17,8 +17,6 @@
  * License-Filename: LICENSE
  */
 
-import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
-
 import java.nio.charset.Charset
 
 @Suppress("DSL_SCOPE_VIOLATION") // See https://youtrack.jetbrains.com/issue/KTIJ-19369.
@@ -28,7 +26,6 @@ plugins {
 
     // Apply third-party plugins.
     alias(libs.plugins.graal)
-    alias(libs.plugins.shadow)
 }
 
 application {
@@ -62,10 +59,6 @@ graal {
 
     mainClass("org.ossreviewtoolkit.cli.OrtMainKt")
     outputName("ort")
-}
-
-tasks.withType<ShadowJar>().configureEach {
-    isZip64 = true
 }
 
 tasks.named<CreateStartScripts>("startScripts").configure {

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -6,7 +6,6 @@ graalPlugin = "0.12.0"
 graphQlPlugin = "6.3.3"
 ideaExtPlugin = "1.1.7"
 kotlinPlugin = "1.8.0"
-shadowPlugin = "7.1.2"
 versionCatalogUpdatePlugin = "0.7.0"
 versionsPlugin = "0.44.0"
 
@@ -71,7 +70,6 @@ graphQl = { id = "com.expediagroup.graphql", version.ref = "graphQlPlugin" }
 ideaExt = { id = "org.jetbrains.gradle.plugin.idea-ext", version.ref = "ideaExtPlugin" }
 kotlin = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlinPlugin" }
 kotlinSerialization = { id = "org.jetbrains.kotlin.plugin.serialization", version.ref = "kotlinPlugin" }
-shadow = { id = "com.github.johnrengelman.shadow", version.ref = "shadowPlugin" }
 versionCatalogUpdate = { id = "nl.littlerobots.version-catalog-update", version.ref = "versionCatalogUpdatePlugin" }
 versions = { id = "com.github.ben-manes.versions", version.ref = "versionsPlugin" }
 

--- a/helper-cli/build.gradle.kts
+++ b/helper-cli/build.gradle.kts
@@ -17,26 +17,17 @@
  * License-Filename: LICENSE
  */
 
-import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
-
 import java.nio.charset.Charset
 
 @Suppress("DSL_SCOPE_VIOLATION") // See https://youtrack.jetbrains.com/issue/KTIJ-19369.
 plugins {
     // Apply core plugins.
     application
-
-    // Apply third-party plugins.
-    alias(libs.plugins.shadow)
 }
 
 application {
     applicationName = "orth"
     mainClass.set("org.ossreviewtoolkit.helper.HelperMainKt")
-}
-
-tasks.withType<ShadowJar>().configureEach {
-    isZip64 = true
 }
 
 tasks.named<CreateStartScripts>("startScripts").configure {

